mod/01_core/set/all/fetch.rb

```diff
@@ -54,8 +54,11 @@ def fetch mark, opts={}
     if card.new_card?
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
+      elsif opts[:skip_virtual]
```

Comment:
Yes, this is better.  The original logic did just return in this case without testing virtual?.  We might look at the :skip_virtual vs. :skip_modules matrix.  I'm not sure we use all the case.  Do we do skip_virtual without skip_modules?  The other way around?